millepede-ii.sh

```diff
@@ -16,13 +18,17 @@ rsync -av --delete --exclude="**/.git" ${SOURCEDIR}/ .
 
 case $ARCHITECTURE in
     osx*)
-    [[ ! $ZLIB_ROOT ]] && ZLIB_ROOT=`brew --prefix zlib` ;;
+    [[ ! $ZLIB_ROOT ]] && ZLIB_ROOT=$(brew --prefix zlib)
+    [[ ! $GFORTRAN ]] && GFORTRAN=gfortran
```

Comment:
What is supposed to set this variable?

Reply:
probably nothing from the outside, I just kept the same code as for other variables.